### PR TITLE
Include official IFC4X1 example files in testdata.

### PR DESCRIPTION
**Pull request by SamuilsRulovs**
I have added in the folder “IFC4x1ExampleFiles”  folder “official”, where are staged all data from the  link below: https://standards.buildingsmart.org/IFC/RELEASE/IFC4_1/FINAL/HTML/link/annex-e.htm
I have staged all files just as how they are staged on the website: 

- [ ] E2_Basic_geometric_shape
- [ ] E3_Advanced_geometric_shape
- [ ] E4_Tessellated_shape
- [ ] E5_Tessellated_shape_with_style
- [ ] E6_Mapped_geometric_shape
- [ ] E7_Placements_and_GIS_referencing
- [ ] E8_Building_element_standard_case
- [ ] E9_Building_element_elemented_case
- [ ] E10_Building_element_configuration
- [ ] E11_Building_service_element
- [ ] E12_Structural_analysis_model
- [ ] E13_Construction_scheduling
- [ ] E14_Additional_test_cases
- [ ] E15_Alignment

### CODE REVIEW

File: testdata/IFC4x1ExampleFiles/official/E10_Building_element_configuration/wall-with-opening-and-window.ifc [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+#4= IFCUNITASSIGNMENT((#15,#16,#17,#18,#19,#20,#21,#22,#23));
+#7= IFCPERSONANDORGANIZATION(#24,#25,$);
+#8= IFCAPPLICATION(#25,'0.10','Test Application','TA 1001');
+#9= IFCAXIS2PLACEMENT3D(#26,$,$);
+#10= IFCDIRECTION((0.,1.));
+#11= IFCSHAPEREPRESENTATION(#3,'Axis','Curve2D',(#27));
+#12= IFCSHAPEREPRESENTATION(#3,'Body','SweptSolid',(#29));
+#13= IFCSHAPEREPRESENTATION(#3,'Body','SweptSolid',(#30));
+#14= IFCSHAPEREPRESENTATION(#3,'Body','SweptSolid',(#32));
+#15= IFCSIUNIT($,.LENGTHUNIT.,.MILLI.,.METRE.);
```
Comment (pjanck) [inline testdata/IFC4x1ExampleFiles/official/E10_Building_element_configuration/wall-with-opening-and-window.ifc]: Most of the example files provided by buildingSMART have a mistake here. This should be corrected for all IFCSIUNIT-s in all files.
```suggestion
#15= IFCSIUNIT(*,.LENGTHUNIT.,.MILLI.,.METRE.);
```

File: testdata/IFC4x1ExampleFiles/official/E15_Alignment/terrain-and-alignment.ifc [hunk 2]
```diff
@@ -0,0 +1,213 @@
...
...4775),(4774,4776,4775),(4777,4779,4778),(4777,4779,4778),(4777,4779,4778),(4780,4782,4781),(4780,4782,4781),(4780,4782,4781),(4783,4785,4784),(4783,4785,4784),(4783,4785,4784),(4786,4788,4787),(4786,4788,4787),(4786,4788,4787),(4789,4791,4790),(4789,4791,4790),(4789,4791,4790),(4792,4794,4793),(4792,4794,4793),(4792,4794,4793),(4795,4797,4796),(4795,4797,4796),(4795,4797,4796),(4798,4800,4799),(4798,4800,4799),(4798,4800,4799),(4801,4803,4802),(4801,4803,4802),(4801,4803,4802),(4804,4806,4805),(4804,4806,4805),(4804,4806,4805),(4807,4809,4808),(4807,4809,4808),(4807,4809,4808),(4810,4812,4811),(4810,4812,4811),(4810,4812,4811),(4813,4815,4814),(4813,4815,4814),(4813,4815,4814),(4816,4818,4817),(4816,4818,4817),(4816,4818,4817),(4819,4821,4820),(4819,4821,4820),(4819,4821,4820),(4822,4824,4823),(4822,4824,4823),(4822,4824,4823),(4825,4827,4826),(4825,4827,4826),(4825,4827,4826),(4828,4830,4829),(4828,4830,4829),(4828,4830,4829),(4831,4833,4832),(4831,4833,4832),(4831,4833,4832),(4834,4836,4835),(4834,4836,4835),(4834,4836,4835),(4837,4839,4838),(4837,4839,4838),(4837,4839,4838),(4840,4842,4841),(4840,4842,4841),(4840,4842,4841),(4843,4845,4844),(4843,4845,4844),(4843,4845,4844)),$);
```
Comment (pjanck) [inline testdata/IFC4x1ExampleFiles/official/E15_Alignment/terrain-and-alignment.ifc:27]: Every triangle is included three times.

File: testdata/IFC4x1ExampleFiles/official/E15_Alignment/terrain-and-alignment.ifc [hunk 3]
```diff
@@ -0,0 +1,213 @@
...
...7286),(7285,7287,7286),(7288,7290,7289),(7288,7290,7289),(7288,7290,7289),(7291,7293,7292),(7291,7293,7292),(7291,7293,7292),(7294,7296,7295),(7294,7296,7295),(7294,7296,7295),(7297,7299,7298),(7297,7299,7298),(7297,7299,7298),(7300,7302,7301),(7300,7302,7301),(7300,7302,7301),(7303,7305,7304),(7303,7305,7304),(7303,7305,7304),(7306,7308,7307),(7306,7308,7307),(7306,7308,7307),(7309,7311,7310),(7309,7311,7310),(7309,7311,7310),(7312,7314,7313),(7312,7314,7313),(7312,7314,7313),(7315,7317,7316),(7315,7317,7316),(7315,7317,7316),(7318,7320,7319),(7318,7320,7319),(7318,7320,7319),(7321,7323,7322),(7321,7323,7322),(7321,7323,7322),(7324,7326,7325),(7324,7326,7325),(7324,7326,7325),(7327,7329,7328),(7327,7329,7328),(7327,7329,7328),(7330,7332,7331),(7330,7332,7331),(7330,7332,7331),(7333,7335,7334),(7333,7335,7334),(7333,7335,7334),(7336,7338,7337),(7336,7338,7337),(7336,7338,7337),(7339,7341,7340),(7339,7341,7340),(7339,7341,7340),(7342,7344,7343),(7342,7344,7343),(7342,7344,7343),(7345,7347,7346),(7345,7347,7346),(7345,7347,7346),(7348,7350,7349),(7348,7350,7349),(7348,7350,7349),(7351,7353,7352),(7351,7353,7352),(7351,7353,7352),(7354,7356,7355),(7354,7356,7355),(7354,7356,7355)),$);
```
Comment (pjanck) [inline testdata/IFC4x1ExampleFiles/official/E15_Alignment/terrain-and-alignment.ifc:29]: Here as well, every triangle is included three times.

File: testdata/IFC4x1ExampleFiles/official/E5_Tessellated_shape_with_style/tessellation-with-image-texture.ifc [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+#52= IFCCARTESIANPOINT((0.,0.,0.));
+#53= IFCSURFACESTYLERENDERING(#55,$,$,$,$,$,$,$,.NOTDEFINED.);
+#54= IFCSURFACESTYLEWITHTEXTURES((#56));
+#55= IFCCOLOURRGB($,1.,1.,1.);
+#56= IFCIMAGETEXTURE(.T.,.T.,'TEXTURE',#57,$,'C:\\dev\\Constructivity\\Constructivity.Renderer.Mono\\Textures\\diagnostic.png');
```
Comment (pjanck) [inline testdata/IFC4x1ExampleFiles/official/E5_Tessellated_shape_with_style/tessellation-with-image-texture.ifc:64]: This path is absolute ...

**Review by pjanck — CHANGES_REQUESTED** [review]
Contact me if comments are not clear.

**Review by hechth — APPROVED** [review]

**Review by pjanck — APPROVED** [review]

**Pull request merged**